x/subscription/spec/01_concepts.md

```diff
@@ -0,0 +1,29 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Subscription
+
+```go
+type Plan struct {
+  id int,  // unique identifier
+  owner Address,
+  price Coin,
+  interval int, // number of days
+}
+
+type Subscription struct {
```

Comment:
How is "expiration" going to be managed -- can merchant / payment gateway cancel/unsubscribe without cancelling the entire plan?

Reply:
merchant cancel some specific subscriber? it can also choose to stop the off-chain service if it can't collect payments from the subscriber, and resume the service as soon as payment resumes.

Reply:
> merchant cancel some specific subscriber?
yes

> it can also choose to stop the off-chain service if it can't collect payments from the subscriber, and resume the service as soon as payment resumes.

that doesn't solve the problem, as it'll keep charging (the collection is per plan) / merchant (or payment gateway on its behalf) can't cancel individual subscriptions

Reply:
both merchant and subscriber can cancel subscription now

Reply:
both plan and subscription has expiration time now.